trojsten/settings/common.py

```diff
@@ -466,6 +466,14 @@ def skip_unreadable_post(record):
     "TROJSTENWEB_SUBMIT_PATH", os.path.join(PROJECT_DIR, PROJECT_MODULE_NAME, "submits")
 )
 SUBMIT_DESCRIPTION_ALLOWED_EXTENSIONS = [".pdf", ".txt", ".md", ".rtf", ".doc", ".docx", ".odt"]
+SUBMIT_DESCRIPTION_ALLOWED_MIMETYPES = [
+    "application/pdf",
+    "text/plain",
+    "text/rtf",
+    "application/msword",
+    "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+    "application/vnd.oasis.opendocument.text",
```

Comment:
mne by sa celkom pacilo mat k tomuto testy so sample suborom kazdeho formatu ci ho zdeteguje spravne..
totiz napr. minimalne pre RTF sa zvyknu pouzivat aj `text/rtf` aj `application/rtf`, a viem si predstavit, ze niekedy v buducnosti vyjde nova verzia `python-magic` ktora zrazu zacne pouzivat ten druhy mime-type, my upgradneme a rozbije sa to

Reply:
jo, testy dopíšem, len neviem, kam môžem slobodne dať takéto sample súbory